packages/react-query/src/shared/hooks/createHooksInternal.tsx

```diff
@@ -606,6 +606,7 @@ export function createRootHooks<
       {
         queries: queries.map((query) => ({
           ...query,
+          queryFn: query.queryFn,
```

Comment:
weirdly enough it doesn't pass without this explicit access on queryFn:

![CleanShot 2024-10-13 at 21 40 35@2x](https://github.com/user-attachments/assets/e1c6374c-fefe-4961-9f10-0b5134c19c28)

Reply:
this is absurd... i think the upstream changes in https://github.com/TanStack/query/pull/8082 is tangling with the type requirement of `queryFn`. 

I struggle to get this to pass in `trpc.useSupenseQueries` when it looks identical to a working `trpc.useQueries`

![CleanShot 2024-10-13 at 23 58 09](https://github.com/user-attachments/assets/d074a232-17ab-455c-9c8d-7381f0e38c7b)

Reply:
it really wants that queryFn here lol...

![CleanShot 2024-10-14 at 00 14 08](https://github.com/user-attachments/assets/afe409b1-eac3-4266-980b-b5e19cdc64d6)

![CleanShot 2024-10-14 at 00 14 42](https://github.com/user-attachments/assets/5c45ac61-c80f-4f1e-a272-db57ec9e63b7)

Reply:
Made an upstream PR : https://github.com/TanStack/query/pull/8176